.github/ISSUE_TEMPLATE/release.md

```diff
@@ -23,8 +23,6 @@ examples of each step, assuming release vX.Y.0 is being cut.
 
 - [ ] Prepared the release branch `release-X.Y` at the beginning of [Code Freeze]:
   - [ ] Created the release branch.
-  - [ ] Created and merged an empty commit to the `master` branch, if required to have it at least one commit ahead of the release branch.
```

Comment:
thanks @jbw976 for the clarification. In that case, perhaps it would make sens to move then this step under the actions done on the code freeze:

```
- [ ] Updated, in a single PR, the following on `master`:
  - [ ] The [releases table] in the `README.md`, removing the now old unsupported release and adding the new one.
  - [ ] The `baseBranches` list in `.github/renovate.json5`, removing the now old unsupported release and adding the new one.
```

In this way, during the code freeze, `renovate` can start creating PRs against the new release branch and we do not need to create an extra empty commit, because we are going to have a real one pushed to the master? wdyt?